src/shogun/machine/LinearMachine.h

```diff
@@ -160,6 +188,8 @@ class CLinearMachine : public CMachine
 		float64_t bias;
 		/** features */
 		CDotFeatures* features;
+		/** bias flag*/
```

Comment:
Please do proper comments. The added information of this one is zero.

/** If true, bias is computed in ::train method */